client/lib/directly/test/index.js

```diff
@@ -36,65 +42,109 @@ describe( 'index', () => {
 		delete require.cache[ require.resolve( '..' ) ];
 	} );
 
-	describe( '#initialize()', () => {
-		it( 'creates a window.DirectlyRTM function', () => {
-			directly.initialize();
-			expect( typeof window.DirectlyRTM ).to.equal( 'function' );
+	describe( 'when the API says Directly is available', () => {
+		useNock( ( nock ) => {
+			nock( 'https://public-api.wordpress.com:443' )
+				.persist()
+				.get( '/rest/v1.1/help/directly/mine' )
+				.reply( 200, {
+					isAvailable: true
+				} );
 		} );
 
-		it( 'attempts to load the remote script', () => {
-			directly.initialize();
-			expect( loadScript.loadScript ).to.have.been.calledOnce;
-		} );
+		describe( '#initialize()', () => {
+			it( 'creates a window.DirectlyRTM function', ( done ) => {
+				directly.initialize()
+					.then( () => expect( typeof window.DirectlyRTM ).to.equal( 'function' ) )
+					.then( () => done() );
+			} );
 
-		it( 'does nothing after the first call', () => {
-			directly.initialize();
-			directly.initialize();
-			directly.initialize();
+			it( 'attempts to load the remote script', ( done ) => {
```

Comment:
As `mocha-promise` having rolled in, we now can simply return the `Promise` instead relying on the `done` callback. e.g. this could be written as:
```
            it( 'attempts to load the remote script', () => (
                directly.initialize()
                    .then( () => expect( loadScript.loadScript ).to.have.been.calledOnce )
            ) );
```

Not a necessary change, just thought you might be interested in knowing :)